### DISCUSSION TITLE
Removed

### DISCUSSION THREAD

**Discussion by ghost** [community/community#43557] [Projects and Issues]
Removed.

**Comment by jfullstackdev** [comment]
try `/images/screenshot2.png`

not very sure unless I'm seeing the full repo

**Comment by auchynnikau** [comment]
Provided code snippet looks valid and image is correctly referenced, the image file is also located in the same repository. But your repository is private so there is no ability to check it manually. So here are a few things you can try to troubleshoot the issue.

1) Consider that images in markdown are displaying only in a preview mode, but not in editing, sometimes people missing this point.

1) Check the branch permissions. If you are using branch protection rules or maintainer privileges, make sure that the branch containing the image file is accessible to the user who need to view it.

2) Try Cleaning your browser cache and reloading the page. Sometimes the image may no be displayed if the browser is showing and old version of the page.

**Reply by auchynnikau** [reply]
It works fine for me, I can see all the images in `notes.md`:

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/33329898/211145860-099cb7ce-338a-4b61-93eb-f993407f081c.png">

**Reply by auchynnikau** [reply]
Cleaning browser cache didn't help?
